modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -1838,6 +1838,14 @@ namespace {
         BGR,
         NV12
     };
+    std::ostream& operator<<(std::ostream& os, TestSourceType a) {
+        os << "Source:";
+        switch (a) {
+            case TestSourceType::BGR:  return os << "BGR";
+            case TestSourceType::NV12: return os << "NV12";
+            default: CV_Assert(false && "unknown TestSourceType");
```

Comment:
I believe it should work, but does it make sense to user `GAPI_Assert`

Reply:
I can recall that there was an agreement to use CV_Assert in tests not to depend on GAPI_Assert implementation, but I might be mistaken. Grep says that there are both CV_ and GAPI_ in our tests, need to align that one day